build/npm/src/install.js

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env node
+
+"use strict"
+
+const fs = require('fs'),
+    request = require('request');
+
+const BASE_URL="https://api.github.com/repos/getgauge/gauge/releases",
+    ARCH_MAPPING = {
+        "ia32": "x86",
+        "x64": "x86_64"
+    },
+    PLATFORM_MAPPING = {
+        "darwin": "darwin",
+        "linux": "linux",
+        "win32": "windows"
+    };
+ 
+var getVersion = function(p) {
+    return new Promise( (resolve, reject) => {
+        if (!fs.existsSync(p)) {
+            reject("Unable to find package.json.");
+        }
+        fs.readFile(p, (err, data) => {
+            if(err) {
+                reject(err);
+            }
+            resolve(JSON.parse(data).version);   
```

Comment:
`fs.readFile` gives `data` as `Buffer` as default. Provide encoding as `UTF-8` or use `toString` on the data.

Reply:
On another thought, we can get rid of reading the file by requireing the `package.json`.

Reply:
ok. will require the `package.json`